cli/cmd/help.go

```diff
@@ -22,7 +22,12 @@ type DefaultHelpFunc func(*cobra.Command, []string)
 // (for example `tt help version`), we try to get the help from it.
 func configureHelpCommand(cmdCtx *cmdcontext.CmdCtx, rootCmd *cobra.Command) error {
 	// Add information about external modules into help template.
-	rootCmd.SetUsageTemplate(fmt.Sprintf(usageTemplate, getExternalCommandsString(&modulesInfo)))
+	if getExternalCommandsString(&modulesInfo) != "  No available external commands" {
+		rootCmd.SetUsageTemplate(fmt.Sprintf(usageTemplateWithExternal,
+			getExternalCommandsString(&modulesInfo)))
```

Comment:
So, we call `getExternalCommandsString`, which loops over modules map, twice. Why?

Reply:
changed